Assignment/solution_student/css/style.css

```diff
@@ -27,15 +27,179 @@ html {
 .clearfix:after {
   clear: both;
 }
-/******************************************
-/* BASE STYLES
-/*******************************************/
+
 body {
-  color: black;
-  font-size: 16px;
+  font-size: 13px;
+  font-weight: lighter;
   background: white;
+  padding: 50px;
```

Comment:
Same as next comment — probably don't need to space it out from the edges really, so you shouldn't need this padding.